src/app.js

```diff
@@ -16,10 +17,20 @@ app.engine(
     extname: "hbs",
     layoutsDir: path.join(__dirname, "views", "layouts"),
     defaultLayout: "main",
-    partialsDir: path.join(__dirname, "views", "partials")
+    // partialsDir: path.join(__dirname, "views", "partials")
   })
 );
-app.use(router)
 
 
+app.get("/", (req, res,next) => {
```

Comment:
This method should be in a controller. Then write the method in the router index.js controller folders.